README.md

```diff
@@ -444,19 +444,13 @@ Some other useful tasks are:
 
 - `./gradlew acceptanceTest`: run the ATs
 - `./gradlew fullAcceptanceTest`: run the ATs in the context of Docker
-- `./gradlew startNakadi`: build Nakadi and start the docker-compose with necessary services (postgresql, zookeeper, kafka)
-- `./gradlew stopNakadi`: shutdown the docker-compose
+- `./gradlew startNakadi`: build Nakadi and start docker-compose services: nakadi, postgresql, zookeeper and kafka
+- `./gradlew stopNakadi`: shutdown docker-compose services
+- `./gradlew startPZK`: start docker-compose services: postgres, zookeeper and kafka (useful for development purposes)
```

Comment:
My personal preference is for "startStorages" since it's more resilient to changes in the underlying technologies.